modules/graceful-shutdown.adoc

```diff
@@ -37,28 +37,27 @@ $ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-s
 ----
 <1> To ensure that the cluster can restart gracefully, plan to restart it on or before the specified date. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.
 
-. Shut down all of the nodes in the cluster. You can do this from your cloud provider's web console, or you can use the below commands:
-
-.. Obtain the list of nodes:
+. Shut down all of the nodes in the cluster. You can do this from your cloud provider's web console, or run the following loop:
 +
 [source,terminal]
 ----
-$ nodes=$(oc get nodes -o jsonpath='{.items[*].metadata.name}')
+$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1; done
 ----
-
-.. Shut down all of the nodes:
 +
-[source,terminal]
+.Example output
 ----
-$ for node in ${nodes[@]}
-do
-    echo "==== Shut down $node ===="
-    ssh core@$node sudo shutdown -h 1
-done
+Starting pod/ip-10-0-130-169us-east-2computeinternal-debug ...
+To use host binaries, run `chroot /host`
+Shutdown scheduled for Mon 2021-09-13 09:36:17 UTC, use 'shutdown -c' to cancel.
+
+Removing debug pod ...
+Starting pod/ip-10-0-150-116us-east-2computeinternal-debug ...
+To use host binaries, run `chroot /host`
+Shutdown scheduled for Mon 2021-09-13 09:36:29 UTC, use 'shutdown -c' to cancel.
 ----
-+
+
```

Comment:
You need to restore these + signs. They make the comment and note render correctly under the step. (Check the preview - the step numbers have reset after your change.)

```suggestion
+
```